tests/test_process_tomography.py

```diff
@@ -1,15 +1,23 @@
-from forest_benchmarking.tomography import proj_to_cp, proj_to_tni, \
-    generate_process_tomography_experiment, acquire_tomography_data, pgdb_process_estimate, \
-    proj_to_tp
-from forest_benchmarking.tomography import _constraint_project
+import networkx as nx
+import numpy as np
+import pytest
+from rpcq.messages import PyQuilExecutableResponse
 
+from forest_benchmarking.compilation import basic_compile
+from forest_benchmarking.random_operators import haar_rand_unitary
 from forest_benchmarking.superop_conversion import vec, unvec, kraus2choi
-from forest_benchmarking.utils import sigma_x, sigma_y, sigma_z, partial_trace
-from numpy import pi
-import numpy as np
-from scipy.linalg import expm
+from forest_benchmarking.tomography import _constraint_project, linear_inv_process_estimate
+from forest_benchmarking.tomography import proj_to_cp, proj_to_tni, \
+    generate_process_tomography_experiment, pgdb_process_estimate, \
+    proj_to_tp
+from forest_benchmarking.utils import sigma_x, partial_trace
 from pyquil import Program
-from pyquil.gates import *
+from pyquil.api import QuantumComputer
+from pyquil.api._compiler import _extract_attribute_dictionary_from_program
+from pyquil.api._qac import AbstractCompiler
+from pyquil.device import NxDevice
+from pyquil.operator_estimation import measure_observables
+from pyquil.pyqvm import PyQVM
 
 REVERSE_CNOT_KRAUS = np.array([[1, 0, 0, 0], [0, 1, 0, 0], [0, 0, 0, 1], [0, 0, 1, 0]])
 CNOT_KRAUS = np.array([[1, 0, 0, 0], [0, 0, 0, 1], [0, 0, 1, 0], [0, 1, 0, 0]])
```

Comment:
We should remove these; they are no longer used, and the labels might confuse someone.